### PR TITLE
feat(filter): cards and accordion hard coded filter

### PR DESCRIPTION
**Pull request by gepifanio**

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 39335954d62430f801b22b6e8e7d49e326b19527

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5d22dea7591ec60008012b8b

File: components/dropdown/index.css [hunk 1]
```diff
@@ -1,7 +1,41 @@
 .dropdown {
   .select {
-    border: 2px solid #094183;
-    height: 2.5rem;
-    width: 100%;
+    display: block;
```
Comment (TimMansell) [inline components/dropdown/index.css]: These changes affect the `in page navigation` component.  We have a similar dropdown called `styled-select` that we could use instead.

https://pattern-lib-unimelb.netlify.com/?selectedKind=Forms&selectedStory=Styled%20select&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

File: components/filter/cards/CardsFilter.vue [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        <button
+          class="filter__button"
+          @click="resetSearch"
+        >Reset all</button>
+      </div>
+    </div>
+
+    <FilterResultsCount
+      :data="dataFiltered.length"
+      class="filter__results" />
```
Comment (TimMansell) [inline components/filter/cards/CardsFilter.vue]: move this class into component

File: components/forms/_select.css [hunk 3]
```diff
@@ -26,6 +26,7 @@
   position: relative;
   background-color: rgb(var(--col-background-primary-white));
   color: rgb(var(--col-brand));
+  border-radius: 4px;
```
Comment (TimMansell) [inline components/forms/_select.css]: use `--br` & update --br to 4px as buttons are going to be updated to use 4px.

**Pull request merged**